enhancements/ingress/gateway-api-with-cluster-ingress-operator.md

```diff
@@ -622,58 +622,178 @@ will be reconciled in any namespaces, removing the need for ServiceMeshMemberRol
+[Gateway merging](https://docs.google.com/document/d/1qj7Xog2t2fWRuzOeTsWkabUaVeOF7_2t_7appe8EXwA/edit?usp=sharing),
+and it is important to review the outcome for any potential security implications.
+
+##### RBAC
+
+OpenShift ships with a set of [default ClusterRoles](https://docs.openshift.com/container-platform/latest/post_installation_configuration/preparing-for-users.html#default-roles_post-install-preparing-for-users)
+to enable users to implement RBAC. Gateway API has its own [RBAC recommendation](https://gateway-api.sigs.k8s.io/concepts/security-model/#rbac)
+defining which personas get write permission for each object. Gateway API users in OpenShift will need the ability to
+implement one of these security models. To support this, the Ingress Operator will add additional RBAC
+permissions for the following default ClusterRoles:
+
+| OpenShift ClusterRole | GatewayClass | Gateway              | xRoute Types         | ReferenceGrant |
+|-----------------------|--------------|----------------------|----------------------|----------------|
+| cluster-admin         | All          | All                  | All                  | All            |
+| admin                 | None         | **Get, List, Watch** | **All**              | None           |
+| edit                  | None         | **Get, List, Watch** | **All**              | None           |
+| view                  | None         | **Get, List, Watch** | **Get, List, Watch** | None           |
+
+For this initial RBAC implementation, write access to Gateways (other than `cluster-admin`) is not included due
+to their complexity and potential security risks. However, users can still create custom roles, such as a
+`Gateway Operator` role, to enable a role with the ability to write Gateways without requiring super-user privileges.
+
+The following diagram maps the personas in Gateway API's [Simple 3 Tier Model](https://gateway-api.sigs.k8s.io/concepts/security-model/#write-permissions-for-simple-3-tier-model) to potential OpenShift ClusterRoles:
+```mermaid
+flowchart LR
+    InfraOperator[Infrastructure Operator] --> ClusterAdmin(cluster-admin)
+    subgraph ClusterRole
+        ClusterAdmin
+        Admin
+        Edit
+    end
+    subgraph Gateway API Persona
+        InfraOperator
+        ClusterOperator
+        AppDev
+    end
+    ClusterOperator[Cluster Operator] --> ClusterAdmin(cluster-admin)
+    AppDev[Application Developer] --> Admin(admin)
+    AppDev[Application Developer] --> Edit(edit)
+```
+
+The Infrastructure Operator is generally responsible for installing and configuring the Gateway API provider
+(e.g., Istio), installing the Gateway API CRDs, and creating the GatewayClass. However, since the Ingress Operator
+handles the installation of OSSM and the CRDs, the Infrastructure Operator is only responsible for creating
+and managing the GatewayClass. Cluster Operators create and manage the Gateways, while Application Developers create
+and manage the routes.
```

Comment:
```suggestion
and manage the routes and referenceGrants.
```

Reply:
This depends on the outcome of https://github.com/openshift/enhancements/pull/1741#discussion_r2006550765